src/riak_cs_gc_key_list.erl

```diff
@@ -175,7 +178,11 @@ gc_index_query(RcPid, EndTime, BatchSize, Continuation, UsePaginatedIndexes) ->
             ok
     end,
 
-    {QueryResult, EndTime}.
+    {QueryResult, EndKey}.
```

Comment:
This `EndKey` may be used in logging when error occurs.
The code looks like:

```
eligible_manifest_keys({{error, Reason}, EndTime}, _) ->
    _ = lager:warning("Error occurred trying to query from time 0 to ~p"
                      "in gc key index. Reason: ~p",
                      [EndTime, Reason]),
    [].
```

If this log line is left as is, `StartKey` is also needed.